Detectors/FIT/FT0/calibration/testWorkflow/FT0ViewObjectsGenerator-Workflow.cxx

```diff
@@ -0,0 +1,35 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
(Calibration meeting conclusion)
So this workflow will be removed. All of visualision will be done on QC side (attaching qc workflows).